sdks/python/test-suites/gradle.properties

```diff
@@ -27,3 +27,7 @@ dataflow_chicago_taxi_example_task_py_versions=3.7
 
 # direct test-suites
 direct_mongodbio_it_task_py_versions=3.5
+
+# portable test-suites
+portable_flink_validates_runner_py_versions=3.6
```

Comment:
Nit: Can you include "postcommit" so it's more clear what is the difference between these two variables?

Optionally: you can remove "portable" since there is no such thing as non-portable Flink Python.

Reply:
Thanks, done.